sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -1037,6 +1037,7 @@ public void listConfigurationSettingsAcceptDateTime(HttpClient httpClient, Confi
      */
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.data.appconfiguration.TestHelper#getTestParameters")
+    @Disabled("https://github.com/Azure/azure-sdk-for-java/issues/37013")
```

Comment:
Does this only fail in live testing? Or when JdkHttpClient is being used? If either of those are true it may be a bug in JdkHttpClient that needs to get fixed.

Reply:
This is a FeatureFlag-related bug. When fixing the feature flag bugs, the configuration setting store accepts the invalid configuration as the feature flag. Then deserialization into FeatureFlag introduces this error.